#>>>>TODO: this needs tunning

attractor_gain: [1,1,1]
repulsive_gain: 2
lambda_repulsive: 10.0
vortex_gain: 0.05
lambda_vortex: 10.0